helmfile.d/values/prometheus-blackbox-exporter-sc.yaml.gotmpl

```diff
@@ -64,10 +64,27 @@ config:
         preferred_ip_protocol: "ip4"
         tls_config:
           insecure_skip_verify: {{ not .Values.global.verifyTls }}
-    {{- if or (eq .Values.harbor.database.type "external") (eq .Values.harbor.redis.type "external") }}
     tcp_connect:
       prober: tcp
-    {{- end }}
+    rabbitmq_amqp:
+      prober: tcp
+      timeout: 10s
+      tcp:
+        tls: false
+        query_response:
+          - send: "AMQP\x00\x00\x09\x01"
+          - expect: "AMQP"
+    rabbitmq_amqps:
+      prober: tcp
+      timeout: 10s
+      tcp:
+        tls: true
+        tls_config:
+          insecure_skip_verify: {{ not .Values.global.verifyTls }}
+        query_response:
+          - send: "AMQP\x00\x00\x09\x01"
+          - expect: "AMQP"
+
```

Comment:
We might want to add secure and insecure variants here to give us some flexibility.